api/seeds/test/apds.js

```diff
@@ -21,58 +21,1576 @@ exports.seed = async knex => {
       status: 'not draft'
     }
   ]);
-
-  await knex('activities').insert([
+  await knex('apd_incentive_payments').insert([
```

Comment:
This seed is basically modified to inject Nick's APD into the test data.  Ideally the test data includes at least one fully-formed APD so we can make assertions about the structure of the data returned by the API.  This should accomplish that.